app/common/directives/collection-toggle/collection-toggle-link.js

```diff
@@ -0,0 +1,22 @@
+module.exports = [
+    '$rootScope',
+    '$translate',
+function (
+    $rootScope,
+    $translate
```

Comment:
Not needed. There are no $translate calls in this file at all. You don't need to inject the service to use the translate directives in your template.